### PR TITLE
Use password field with toggle to set PIN

### PR DESCRIPTION
**Pull request by eliykat**
## Objective

Desktop implementation of https://github.com/bitwarden/browser/pull/1911. They're identical, so I'll duplicate any code review changes across both PRs.

### CODE REVIEW

File: src/app/accounts/settings.component.ts [hunk 1]
```diff
@@ -198,11 +200,27 @@ export class SettingsComponent implements OnInit {
...
 
             div.innerHTML =
-                `<div class="swal2-text">${this.i18nService.t('setYourPinCode')}</div>` +
-                '<input type="text" class="swal2-input" id="pin-val" autocomplete="off" ' +
-                'autocapitalize="none" autocorrect="none" spellcheck="false" inputmode="verbatim">';
+                `<div class="swal2-text">${this.i18nService.t('setYourPinCode')}</div>
+                <div class="swal2-password-input-group">
+                    <input type="password" class="swal2-input" id="pin-input" autocomplete="off"
+                        autocapitalize="none" autocorrect="none" spellcheck="false" inputmode="verbatim"
+                        placeholder="${this.i18nService.t('pin')}">
```
Comment (Hinton) [inline src/app/accounts/settings.component.ts:207]: Sanitize.

**Review by Hinton — CHANGES_REQUESTED** [review]
We need to sanitize any dynamic value in the `innerHTML`.

**Comment by eliykat** [comment]
Waiting to see if I can use an Angular component instead, per bitwarden/jslib#417

**Comment by eliykat** [comment]
Closing this, will reopen new PR using dynamic modals.

**Pull request closed without merging**